src/mlpack/tests/cv_test.cpp

```diff
@@ -66,20 +64,22 @@ BOOST_AUTO_TEST_CASE(BinaryClassificationMetricsTest)
 
   LogisticRegression<> lr(data, predictedLabels);
 
-  BOOST_REQUIRE_CLOSE(Accuracy::Evaluate(lr, data, labels), 0.7, 1e-5);
+  REQUIRE(Accuracy::Evaluate(lr, data, labels) == Approx(0.7).epsilon(1e-7));
 
-  BOOST_REQUIRE_CLOSE(Precision<Binary>::Evaluate(lr, data, labels), 0.6, 1e-5);
+  REQUIRE(Precision<Binary>::Evaluate(lr, data, labels)
+          == Approx(0.6).epsilon(1e-7));
```

Comment:
This should be tabbed twice (4 spaces), see https://github.com/mlpack/mlpack/wiki/DesignGuidelines#line-length-and-wrapping from more details.

The same comment applies to some other lines below as well.